assignments/objects.js

```diff
@@ -18,28 +18,73 @@ const example = {
 }
 
 // Write your intern objects here:
+let intern1 = {
```

Comment:
The names of the people would have been a better choice for the variable name as it is easier to refer to them later.